package.json

```diff
@@ -36,6 +36,7 @@
     "js-yaml": "~3.6.1",
     "marked": "~0.3.5",
     "marked-terminal": "~1.6.1",
+    "mkdirp": "^0.5.1",
```

Comment:
Have a look at the `fs-extra` module already included. It has this functionality. From the top of my mind it's called using `plugins.fs`